client/sites-dashboard/controller.tsx

```diff
@@ -2,8 +2,8 @@ import { Global, css } from '@emotion/react';
 import { SitesDashboard } from './components/sites-dashboard';
 import type { Context as PageJSContext } from 'page';
 
-const globalStyles = css`
-	body.is-group-sites-dashboard {
+export const globalStyles = css`
+	body.is-group-sites {
```

Comment:
I was worried that having our dashboard in the existing "sites" group would mean the styles we add to `body.is-group-sites` would pollute existing views that are already in that group.

But I just realised that the `<Global>` component from `@emotion/react` [will remove the global styles when our view gets unmounted](https://emotion.sh/docs/globals). So using this section is going to be fine.

Reply:
![CleanShot 2022-07-15 at 16 09 42@2x](https://user-images.githubusercontent.com/1500769/179148629-61763ecb-dc9a-49eb-94e9-414da409280b.png)

This extra padding you see if you access the dashboard via `/sites-dashboard` can be tidied up if use the other group selector too.

```suggestion
	body.is-group-sites, body.is-group-sites-dashboard {
```

Reply:
Thanks for the good TIL. In the final solution, I left `is-group-sites-dashboard` as is.